src/CommunityToolkit.Maui.Core/Handlers/CameraView/CameraManager.ios.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System.Runtime.InteropServices;
+using AVFoundation;
+using CommunityToolkit.Maui.Core.Primitives;
+
+namespace CommunityToolkit.Maui.Core.Handlers;
+
+public partial class CameraManager
+{
+	AVCaptureSession? captureSession;
+	AVCapturePhotoOutput? photoOutput;
+
+	internal Action? Loaded { get; set; }
+
+	// IN the future change the return type to be an alias
+	public UIView CreatePlatformView()
+	{
+		captureSession = new AVCaptureSession
+		{
+			SessionPreset = AVCaptureSession.PresetPhoto
+		};
+
+		photoOutput = new AVCapturePhotoOutput();
+
+		var previewView = new PreviewView();
+		previewView.Session = captureSession;
+
+		return previewView;
+	}
+
+	public void Dispose()
+	{
+		Dispose(true);
+		GC.SuppressFinalize(this);
+	}
+
+	protected virtual partial void PlatformConnect()
+	{
+		var device = AVCaptureDevice.GetDefaultDevice(AVMediaTypes.Video) ?? throw new InvalidOperationException();
+
+		ArgumentNullException.ThrowIfNull(captureSession);
+		ArgumentNullException.ThrowIfNull(photoOutput);
+
+		captureSession.AddInput(new AVCaptureDeviceInput(device, out var err));
+		captureSession.AddOutput(photoOutput);
+		captureSession.StartRunning();
+	}
+
+	protected virtual partial void PlatformDisconnect()
+	{
+	}
+
+	protected virtual partial void PlatformTakePicture()
+	{
+		ArgumentNullException.ThrowIfNull(photoOutput);
+
+		var wrapper = new AVCapturePhotoCaptureDelegateWrapper();
+		var avCapturePhotoSettings = AVCapturePhotoSettings.Create();
+
+		AVCapturePhotoSettings.FromFormat(new NSDictionary<NSString, NSObject>(new[] {AVVideo.CodecKey},
+			new[] {(NSObject) new NSString("jpeg")}));
+
+		photoOutput.CapturePhoto(avCapturePhotoSettings, wrapper);
+
+		Task.Run(async () =>
+		{
+			var (output, photo, error) = await wrapper.Task;
+
+			var nsData = photo.FileDataRepresentation;
+
+			if (nsData == null)
```

Comment:
We use pattern matching for null checks
```suggestion
			if (nsData is null)
```